gtp_cmd_handlers.py

```diff
@@ -300,6 +282,36 @@ def _tree_search_wrapper(self, parallel_readouts=None):
                 self._last_report_time = now
         return leaves
 
+    def _minigui_report_position(self):
```

Comment:
Is this behind a flag so that it doesn't happen in normal GTP mode?

Reply:
yes